cpp/src/ray/runtime/abstract_ray_runtime.cc

```diff
@@ -199,6 +199,15 @@ std::string AbstractRayRuntime::CallActor(
     func_holder.class_name = typed_descriptor->ClassName();
     invocation_spec = BuildInvocationSpec1(
         TaskType::ACTOR_TASK, func_holder, args, ActorID::FromBinary(actor));
+  } else if (remote_function_holder.lang_type == LangType::JAVA) {
```

Comment:
Can we avoid the duplicated code here?

Reply:
Only two lines can be re-used, maybe it's better to keep current structure to save one more indent.